pkgs/applications/graphics/litemdview/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, fetchpatch
+, gtkmm3
+, autoconf
+, automake
+, libtool
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "litemdview";
+  # litemdview -v
+  version = "0.0.32";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = "g0tsu";
+    repo = "litemdview";
+    rev = "litemdview-0.0.32";
+    hash = "sha256-XGjP+7i3mYCEzPYwVY+75DARdXJFY4vUWHFpPeoNqAE=";
+  };
+
+  buildInputs = [
+    gtkmm3
+  ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    pkg-config
+    libtool
+  ];
+
+  preConfigure = "./bootstrap";
```

Comment:
What about trying autoreconfHook?

Reply:
Indeed, `autoreconfHook` would be more idiomatic. That's all the bootstrap script seems to do, so we can drop it and remove its dependencies from `nativeBuildInputs`.

Reply:
Done, thanks!